helpers/movimientosArkaHelper/movimientosArkaHelper.go

```diff
@@ -43,37 +42,70 @@ func GetAllEstadoMovimiento(nombre string) (estado []*models.EstadoMovimiento, o
 	return resEstadoMovimiento, nil
 }
 
-func GetAllElementosMovimiento(query string) (estado []*models.ElementosMovimiento, outputError map[string]interface{}) {
+func GetAllElementosMovimiento(query string) (elementos []*models.ElementosMovimiento, outputError map[string]interface{}) {
 
 	defer func() {
 		if err := recover(); err != nil {
-			outputError = map[string]interface{}{"funcion": "/GetAllElementosMovimiento", "err": err, "status": "502"}
+			outputError = map[string]interface{}{"funcion": "/GetAllElementosMovimiento", "err": err, "status": "500"}
 			panic(outputError)
 		}
 	}()
 
-	var (
-		resMovimiento []*models.ElementosMovimiento
-		urlcrud       string
-	)
+	urlcrud := "http://" + beego.AppConfig.String("movimientosArkaService") + "elementos_movimiento?" + query
+	if err := request.GetJson(urlcrud, &elementos); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetAllElementosMovimiento - request.GetJson(urlcrud, &elementos)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+	return elementos, nil
+}
 
-	urlcrud = "http://" + beego.AppConfig.String("movimientosArkaService") + "elementos_movimiento?" + query
-	fmt.Println(urlcrud)
-	if err := request.GetJson(urlcrud, &resMovimiento); err != nil || len(resMovimiento) == 0 {
-		status := "502"
-		if err == nil {
-			err = errors.New("len(resMovimiento) == 0")
-			status = "404"
+func GetAllSoporteMovimiento(query string) (soportes []*models.SoporteMovimiento, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{"funcion": "/GetAllSoporteMovimiento", "err": err, "status": "500"}
+			panic(outputError)
+		}
+	}()
+
+	urlcrud := "http://" + beego.AppConfig.String("movimientosArkaService") + "soporte_movimiento?" + query
+	if err := request.GetJson(urlcrud, &soportes); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetAllSoporteMovimiento - request.GetJson(urlcrud, &soportes)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+	return soportes, nil
+}
+
+func GetAllMovimiento(query string) (movimientos []*models.Movimiento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetAllMovimiento hace tal cosa"
```suggestion
// GetAllMovimiento ...
func GetAllMovimiento(query string) (movimientos []*models.Movimiento, outputError map[string]interface{}) {
```